boards/fox/include/periph_conf.h

```diff
@@ -39,6 +39,9 @@ extern "C" {
 #define CLOCK_AHB_DIV       RCC_CFGR_HPRE_DIV1      /* AHB clock -> 72MHz */
 #define CLOCK_APB2_DIV      RCC_CFGR_PPRE2_DIV1     /* APB2 clock -> 72MHz */
 #define CLOCK_APB1_DIV      RCC_CFGR_PPRE1_DIV2     /* APB1 clock -> 36MHz */
+/* resulting bus clocks */
+#define CLOCK_APB1          (CLOCK_CORECLOCK / 2)
```

Comment:
If I see it correctly, macros in l39-41 aren't used anywhere thus can be removedas well. Having a note for the division by two (like resulting clock frequencies som lines above) should fit.